### PR TITLE
homework lesson-6

### PR DESCRIPTION
**Pull request by DenisKishylia**
Doing 6 homework. Adding the layout of the upper part of the site

### CODE REVIEW

File: Triphouse-project/style.css [hunk 1]
```diff
@@ -33,10 +33,110 @@
     font-family: "Roboto";
   }
 
+  .header {
+    max-width: 1440px;
+    height: 100vh;
+    margin: 0px auto;
+    background:#10367C url(./img/castelmezzano-1979546\ 1.svg) no-repeat;
+    background-attachment: fixed;
```
Comment (BubentsovKirill) [inline Triphouse-project/style.css:41]: Add background-size: cover;

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -8,7 +8,61 @@
...
+                <p>
+                  <img src="./img/google-play-badge.svg" alt="google play market" />
+                </p>
+                <p>
+                  <img src="./img/Download_on_the_App_Store_Badge_US-UK_RGB_blk_092917 1.svg" alt="appStore" />
+                </p>
+              </div>
+            </div>
+    </header>
+    <!-- <main class="offer">
```
Comment (BubentsovKirill) [inline Triphouse-project/index.html:65]: Why is it commented?

File: Triphouse-project/style.css (hunk 1 above)
Comment (DenisKishylia) [reply Triphouse-project/style.css:41]: done

File: Triphouse-project/index.html (hunk 2 above)
Comment (DenisKishylia) [reply Triphouse-project/index.html]: because when I worked I commented out, to make it more convenient, I forgot back.

**Pull request merged**